esphome/components/thermostat/climate.py

```diff
@@ -800,6 +800,11 @@ async def to_code(config):
             var.get_heat_mode_trigger(), [], config[CONF_HEAT_MODE]
         )
         cg.add(var.set_supports_heat(True))
+    if CONF_HEAT_COOL_MODE in config:
+        await automation.build_automation(
+            var.get_heat_cool_mode_trigger(), [], config[CONF_HEAT_COOL_MODE]
+        )
+    cg.add(var.set_supports_heat_cool(heat_cool_mode_available))
```

Comment:
Should be indented more and made static?
```suggestion
        cg.add(var.set_supports_heat_cool(True))
```

Reply:
Works for me 🙂